docs/src/dev-manual/archivematica.md

```diff
@@ -20,34 +20,43 @@ inside the cluster, and they are not tracked in the repository.
 
 ### Quick Checklist for Configuration Files
 
-- [ ] .am.secret File:
-  - Location: `hack/kube/overlays/dev-am/.am.secret`
-  - **Contents to Check:**
-    - AM API address (e.g.,`http://host.k3d.internal:62080`)
-    - User credentials(`user=test`, `api_key=test`)
-    - SFTP configuration
-    details (`sftp_host=`, `sftp_port=`, `sftp_user=`,
-    `sftp_remote_dir=`, `sftp_private_key_passphrase=`).
-- [ ] .id_ed25519.secret File:
-  - Location: `hack/kube/overlays/dev-am/.id_ed25519.secret`
-  - **Contents to Check:**
-    - SSH private key (Ensure it starts with `-----BEGIN
-    OPENSSH PRIVATE KEY-----` and ends with `-----END
-    OPENSSH PRIVATE KEY-----`)
-- [ ] .known_hosts.secret File:
-  - Location: `hack/kube/overlays/dev-am/.known_hosts.secret`
-  - **Contents to Check:**
-    - Known hosts entries (Look for entries starting with
-    `|1|` and containing `ssh-rsa`, `ecdsa-sha2-nistp256`,
-    `ssh-ed25519` etc.)
-- [ ] .tilt.env File:
-  - Location: `root/`
-  - **Contents to Check:**
-    - ENDURO_PRES_SYSTEM = "am"
-- [ ] enduro.toml File:
-  - Location: `root/`
-  - **Contents to Check:**
-    - Preservation Taskqueue variable must be set to "am"
+#### .am.secret File
+
+- Location: `hack/kube/overlays/dev-am/.am.secret`
+- **Contents to Check:**
+  - AM API address (e.g.,`http://host.k3d.internal:62080`)
+  - User credentials(`user=test`, `api_key=test`)
+  - SFTP configuration
+  details (`sftp_host=`, `sftp_port=`, `sftp_user=`,
+  `sftp_remote_dir=`, `sftp_private_key_passphrase=`).
+
+#### .id_ed25519.secret File
+
+- Location: `hack/kube/overlays/dev-am/.id_ed25519.secret`
+- **Contents to Check:**
+  - SSH private key (Ensure it starts with `-----BEGIN
+  OPENSSH PRIVATE KEY-----` and ends with `-----END
+  OPENSSH PRIVATE KEY-----`)
+
+#### .known_hosts.secret File
+
+- Location: `hack/kube/overlays/dev-am/.known_hosts.secret`
+- **Contents to Check:**
+  - Known hosts entries (Look for entries starting with
+  `|1|` and containing `ssh-rsa`, `ecdsa-sha2-nistp256`,
+  `ssh-ed25519` etc.)
```

Comment:
All lists in this PR seem to share the same problem with nesting lists, but I'm using this section as the example. It renders as follows:

![image](https://github.com/artefactual-sdps/enduro/assets/606459/3e914d15-8e7c-476b-9ae2-b678fc783d00)

Given how you indented the code I suspect that wasn't your intention. I believe that the list items under "Contents to check" need to use 4-space indentation in mkdocs. After the change:

```suggestion
#### .known_hosts.secret File

- Location: `hack/kube/overlays/dev-am/.known_hosts.secret`
- **Contents to Check:**
    - Known hosts entries (Look for entries starting with
      `|1|` and containing `ssh-rsa`, `ecdsa-sha2-nistp256`,
      `ssh-ed25519` etc.)
```

It renders as follows:

![image](https://github.com/artefactual-sdps/enduro/assets/606459/460c82e7-c410-41d8-a660-13144cce0e7a)

Reply:
I will check markdownlint to see if this doesn't throw errors. That's my initial worry with this.